src/main/scala/mesosphere/marathon/Main.scala

```diff
@@ -29,6 +29,13 @@ class MarathonApp(args: Seq[String]) extends AutoCloseable with StrictLogging {
     Runtime.getRuntime.asyncExit(CrashStrategy.UncaughtException.code)
   })
 
+  if (LibMesos.isCompatible) {
+    logger.info(s"Successfully loaded libmesos: version ${LibMesos.version}")
+  } else {
+    logger.error(s"Failed to load libmesos: ${LibMesos.version}")
+    System.exit(1)
```

Comment:
What do you think about using a new suicide codes? That makes debugging much simpler.

Reply:
I think it's something we can consider for a separate PR. On that note, I do think this is more of a refusal to start rather than a suicide.